cli/bin/cli.js

```diff
@@ -4,18 +4,21 @@ const open = require('open');
 const { showHelp, showVersion, showErrorMsg } = require('../index');
 
 async function main() {
+  const validFlags = ['-v', '--version', '-h', ' --help'];
   const message = process.argv[2];
 
-  if (message === undefined || message === '-h' || message === '--help') {
+  if (
+    message === undefined ||
+    message.toLowerCase() === '-h' ||
+    message.toLowerCase() === '--help'
+  ) {
```

Comment:
If you'll use `message` just being "lower-cased", I think that you can do it:

```
const message = process.argv[2]?.toLowerCase();
```

Reply:
Yeah, I thought of doing it this way, but it makes the message the user wants to tweet also lowercase.
Here is a test to `node cli.js "How can I learn Relay?"`

![image](https://user-images.githubusercontent.com/64330605/149831320-c44cf8eb-30bb-4564-b940-127fb5ebf681.png)

This `.toLowerCase()` is just to call the help or version functions even if the user type a valid flag in uppercase.

Reply:
Oh, I see! So, I think that you can run something like it, based on the @daniloab's comment that he mentions the *early return*:

```js
if (message.charAt(0) === '-' && !validFlags.includes(message)) {
  // ...
}

switch (message.toLowerCase()) {
  case '-v':
  case '--version':
    showVersion();
    break;
  // Others cases...
}
```

What do you think about it?